examples/auth/app/users/mutations/login.ts

```diff
@@ -0,0 +1,23 @@
+import db from "db"
+import {AuthenticationError} from "blitz"
+import {SessionContext} from "@blitzjs/server"
+
+type LoginInput = {
+  email: string
+  password: string
+}
+
+type Ctx = {session?: SessionContext}
+
+export default async function login({email, password}: LoginInput, ctx: Ctx = {}) {
+  const user = await db.user.findOne({where: {email}})
+
+  if (!user || password !== user.hashedPassword) {
```

Comment:
@flybayer what's the plan for unhashing the password here? Would it be hashed on the client before it's passed as a param in mutation? This is looking great, btw. Authentication & Authorization is something that always seems to drive me crazy.

Reply:
Hashing will be done right here. Just haven't got to that part yet :D